galsim/chromatic.py

```diff
@@ -1359,3 +1770,76 @@ def withScaledFlux(self, flux_ratio):
             return ChromaticAutoCorrelation( self.obj * math.sqrt(flux_ratio), **self.kwargs )
         else:
             return ChromaticObject(self).withScaledFlux(flux_ratio)
+
+class ChromaticOpticalPSF(ChromaticObject):
+    """A subclass of ChromaticObject meant to represent chromatic optical PSFs.
+
+    Chromaticity plays two roles in optical PSFs. First, it determines the diffraction limit, via
+    the wavelength/diameter factor.  Second, aberrations such as defocus, coma, etc. are typically
+    defined in physical distances, but their impact on the PSF depends on their size in units of
+    wavelength.  Other aspects of the optical PSF do not require explicit specification of their
+    chromaticity, e.g., once the obscuration and struts are specified in units of the aperture
+    diameter, their chromatic dependence gets taken care of automatically.
+
+    When using interpolation to speed up image rendering (see ChromaticObject.setupInterpolation()
+    method for details), including ~10-15 samples across any given bandpass should be sufficient.
+    For moderate accuracy, 5 is often sufficient.  Likewise, for high accuracy, `oversample_fac`
+    should be in the range 1.5-2, whereas for moderate accuracy, 1 is often sufficient.  All of
+    these statements assume that aberrations are not very large (typically <~0.25 waves, which is
+    commonly satisfied by space telescopes); if they are larger than that, then more stringent
+    settings are required.
+
+    Also, note that in order to render an image of a real physical object, the ChromaticOpticalPSF
+    should be convolved with some object that has an SED.  Drawing without doing so corresponds to
+    using a flat SED.
```

Comment:
Now I'm even more confused about what your point is in this paragraph.  I thought you were using SED loosely in the sense of having some kind of chromatic dependence, not that there was an `SED` attribute.

I thought you were just making the point that if you convolve a chromatic PSF with a non-chromatic galaxy, then the galaxy implicitly has a flat SED.  The same way that if you convolve a `ChromaticAtmosphere` with `Sersic` then the galaxy implicitly has a flat SED.  Likewise if you convolve a `Kolmogorov` with `Sersic * sed` then the PSF implicitly has a flat SED.

So this paragraph seemed like stuff that users of the various chromatic classes should already understand.  But if you think it's a point worth emphasizing, I think this paragraph should be moved to the main `ChromaticObject` doc string.

Unless you are making some other point that I'm not getting right now.  Is there some reason that `ChromaticOpticalPSF` convolved with `Sersic` behaves qualitatively differently than `Kolmogorov * sed` convolved with `Sersic`?  I would think in both cases, you get the right effective PSF over the provided bandpass convolved with a flat-spectrum galaxy.